lib/src/generator/templates.renderers.dart

```diff
@@ -0,0 +1,149 @@
+// ignore_for_file: camel_case_types, unused_element
```

Comment:
Can we call out generated files in a comment at the beginning?  Something something do not modify, etc.

Reply:
Doh, good call. Done!